advanced/custom/js.mdx

```diff
@@ -11,6 +11,8 @@ Custom JS allows you to add custom executable code globally. It is the equivalen
 
 ## Adding script.js
 
+To add custom JavaScript functionality globally, you need to create a `script.js` file at the root of your project. In case of a monorepo, this file should be at the same level as your `mint.json` file.
```

Comment:
iirc, we'll actually pick up `.js` files nested anywhere in the content directory (path of `mint.json`) or lower

Reply:
Interesting. [I just tested and this is true](https://github.com/mxbaylee/mintlify-starter/commit/a287c98f255b3f251bd5c09c280a88e80a81f316). Are we sure this is the behavior we want? 😅 


I'll update the PR.

Reply:
> Are we sure this is the behavior we want? 😅

long term, probably not haha

Reply:
Updated, what do you think?

Reply:
lgtm!